docs/definingworkflows.rst

```diff
@@ -365,7 +393,59 @@ To ease composability and avoid unwanted collisions, each Stage is defined withi
 Composition using Subworkflows
 ------------------------------
 
+Subworkflows facilitates the definition of an entire workflow to run as a single step inside another workflow. Nesting workflows are very useful, especially when sections of code are run in parallel and are executed independently from each other. 
+
+One example is a workflow that defines two unique steps, the second depending on the first. The second step expects a file from the first step as input, and it will wait until everything in step one is finished before starting. The first step is called as a multistep-stage and so a number of instances are scheduled of the same packtivity. Without a subworfklow the second step would have to wait until all instances are done in step one before proceeding, despite only depending on the output from one of them. 
```

Comment:
Cool!! It's fantastic that you give a concrete example to motivate the use of subworkflows, b/c indeed this benefit isn't immediately obvious (at least it wasn't to me... :P). My suggested mods are just to clarify something I was confused about initially, which is that the scenario of running the first step as a multi-step stage is a hypothetical one (i.e. not what's actually done in the example `workflow.yml`) described to illustrate the benefit of running the entire subworkflow as multi-step instead. However, I may have misunderstood the scenario, so please let me know if anything I've added is misrepresentative!

```suggestion
One example is a workflow that defines two unique steps, the second depending on the first. The second step expects a file from the first step as input, and will wait until everything in step one is finished before starting. Suppose that there are multiple inputs to the first step, each of which can be processed through the entire two-step workflow independently. The workflow execution could be parallelized by running both steps as multi-step stages, where a single instance of step one is scheduled to process each input, and likewise a single instance of step two processes the output from an instance of step one. With this approach, the second step instances would need to wait until all instances of step one are finished before proceeding, despite only depending on the output from one of them. 
```

Reply:
You're definitely right! I was struggling finding the words how to describe the scenario. You did it perfectly though, thank you! :)